.github/buildomat/jobs/deploy.sh

```diff
@@ -139,6 +139,9 @@ fi
 #
 pfexec /sbin/zfs create -o mountpoint=/zone rpool/zone
 
+# Ensure we have the latest chrony package
+pfexec pkg update chrony
```

Comment:
This doesn't change the version of chrony installed in our production images, does it? I thought this was only for our lab tests (still worthwhile keeping up to date, admittedly!).

Reply:
The production images already have chrony 4.5, but buildomat images don't yet, and that's what I was seeing if I could work around for testing this. It doesn't work as pkg is not in the image.

Reply:
This is done now, FYI.